rootfs-builder/alpine/Dockerfile.in

```diff
@@ -10,9 +10,31 @@ RUN apk update && apk add \
     make \
     bash \
     gcc \
+    g++ \
+	wget \
```

Comment:
Strange indent "wobble" - tabs/spaces?

Again, sorting this list alphabetically would help a bit.

Reply:
a command-line helps you guys on this (for mac with gnu sed (i.e. `gsed`)):

```bash
$ pbpaste|tr -d ' \t\'|sort|gsed -e 's/\(.*\)$/\t\1 \\/' -e '$s/\\//'
	apk-tools-static \
	autoconf \
	automake \
	bash \
	binutils \
	cmake \
	coreutils \
	curl \
	g++ \
	gcc \
	git \
	libc-dev \
	libseccomp \
	libseccomp-dev \
	linux-headers \
	m4 \
	make \
	musl \
	musl-dev \
	tar \
	vim \
	wget
```